hw3/01-characters.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>01 - Characters</title>
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+    <link rel="stylesheet" href="styles.css" />
+    <script defer src="01-characters.js"></script>
+  </head>
+  <body>
+    <main>
+      <h1>Exercise 01 - Characters</h1>
+      <section id="results" class="text-center">
+        <img
+          id="loading"
+          class="mb-5"
+          src="loader.gif"
+          alt="gif of a loading element"
+          width="50px"
```

Comment:
Can you elaborate on this image tag? I'm unsure of its purpose and can't find it when inspecting the code in the browser.

Reply:
Looks like an attempt to display a loading gif, however when I pulled down your branch there was no loader gif, so it doesn't show up on this branch at the moment. You might want to push the `loader.gif' to this branch for it to work.